content/en/software_catalog/_index.md

```diff
@@ -152,7 +152,7 @@ The services and resources statistics, and span summaries on the **Service List*
 [1]: https://app.datadoghq.com/services
 [2]: /account_management/rbac/
 [3]: /account_management/rbac/permissions/
-[4]: /developers/guide/data-collection-resolution-retention/
+[4]: /developers/guide/data-collection-resolution/
```

Comment:
```suggestion
[4]: /data_security/data_retention_periods/
```
The link text here is "Learn more about data retention for APM," so I would have expected it to go to your new retention page?